axum-debug/CHANGELOG.md

```diff
@@ -9,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **breaking:** Removed `debug_router` macro.
 - **breaking:** Removed `check_service` function.
 - **breaking:** Removed `debug_service` function.
+- **breaking:** No longer re-exports `axum`.
```

Comment:
Version `0.1.0` does not re-export `axum`. I did re-export change while moving the crate in.